### ISSUE TITLE
scheduler should try to be non-interruptive

### ISSUE THREAD

**Issue by wwalker** [wwalker/remindme#5]
By monitoring `get-idle-time` or similar, the scheduler should wait for at least Y seconds (Y = 5 ?) of inactivity to avoid grabbing keyboard focus, interrupting concentration.  However, if after Z seconds (Z = 30?) of waiting for inactivity and not finding any, interrupt anyway.  Later X and Y should be made configurable (ENV vars and/or command line arguments to the scheduler).
